passes/cmds/setundef.cc

```diff
@@ -243,6 +243,8 @@ struct SetundefPass : public Pass {
 			{
 				for (auto *cell : module->selected_cells()) {
 					for (auto &parameter : cell->parameters) {
+						if (parameter.second.flags & RTLIL::CONST_FLAG_STRING)
+							continue;
```

Comment:
I know `parameter.second.flags & RTLIL::CONST_FLAG_STRING` implies there shouldn't be any undef bits, but this makes the code harder to read for little gain

Reply:
If this isn't done, then bits() is iterated over. In my current implementation, that means unpacking. I don't think this is a readability problem, maybe I'm blind to that with things I wrote myself, but "skip these because they're not interesting anyway" is a common pattern that's easy to recognize for me

Reply:
Yes, but we don't care about unpacking for parameters